torch/quantization/fx/quantize.py

```diff
@@ -78,7 +80,7 @@
 def insert_observer(
         node: Node, observer: torch.quantization.ObserverBase,
         model: torch.nn.Module,
-        activation_post_process_map: Dict[str, torch.quantization.ObserverBase],
+        activation_post_process_map: Dict[str, List[torch.quantization.ObserverBase]],
```

Comment:
does the graph look like

```
          --> obs2
         /
node --> obs1
```

or 

```
node -> obs1 -> obs2
```

?

Reply:
the second one